### PR TITLE
Add snapshot related messages

### PR DESCRIPTION
**Pull request by meghnasiingh**
Signed-off-by: Meghna Singh <Meghna.Singh1@ibm.com>

### CODE REVIEW

**Review by sameshai — APPROVED** [review]
LGTM

File: pkg/messages/messages_en.go [hunk 1]
```diff
@@ -95,6 +119,12 @@ var messagesEn = map[string]Message{
 		Type:        codes.InvalidArgument,
 		Action:      "Please provide volume ID for attach/detach or delete it",
 	},
+	EmptySnapshotID: {
+		Code:        EmptySnapshotID,
+		Description: "napshotID must be provided",
```
Comment (skorati) [inline pkg/messages/messages_en.go:124]: napshotID -> snapshotID

Comment (prankulmahajan) [inline pkg/messages/messages_en.go:124]: `SnapshotID must be provided`

**Review by prankulmahajan — CHANGES_REQUESTED** [review]

**Review by skorati — COMMENTED** [review]
Only one minor change, else looks good.

**Review by sameshai — CHANGES_REQUESTED** [review]
Fix the typos

**Review by sameshai — CHANGES_REQUESTED** [review]
Fix the typos

**Review by sameshai — CHANGES_REQUESTED** [review]
Fix the typos

**Review by GunaKKIBM — APPROVED** [review]

**Review by prankulmahajan — APPROVED** [review]
lgtm

**Review by skorati — APPROVED** [review]
looks good.

**Review by sameshai — APPROVED** [review]

**Pull request merged**